regulations/static/regulations/css/less/module/comment.less

```diff
@@ -316,16 +316,54 @@ Write Mode
     }
   }
 
+  .comment-context-wrapper {
+    border-radius: 2px;
+    box-shadow: 0 0 1px 1px @text_area_border;
+    margin: 30px 0;
+    position: relative;
+  }
+
+  .comment-context-toggle {
+    background-color: @grey;
+    border-radius: 2px;
+    cursor: pointer;
+    font-size: 16px;
+    padding: 10px 15px;
+
+    .comment-context-text {
+      color: @blue;
+    }
+
+    .fa-minus-circle,
+    .comment-context-text-less {
+      display: none;
+    }
+
+    .fa {
+      margin-right: 3px;
+    }
+
+    .comment-context-label {
+      font-weight: bold;
+    }
+
+    .comment-context-type {
+      text-transform: capitalize;
```

Comment:
Looks like this isn't a safe bet:
<img width="669" alt="screen shot 2016-06-02 at 3 25 02 pm" src="https://cloud.githubusercontent.com/assets/326918/15757959/3446f742-28d6-11e6-908b-4f2934c6be8d.png">